lhotse/augmentation/utils.py

```diff
@@ -1,4 +1,9 @@
+from typing import List
+
+import numpy as np
 import torch
+from torchaudio.functional import highpass_biquad
+from torchaudio.transforms import Resample
```

Comment:
Can you remove torchaudio imports from the global namespace into the local function you added? They tend to crash documentation builds last time I checked, because they tried to load some shared library that is not available in that env.

Also, I suggest importing `get_or_create_resampler` from `lhotse.augmentation.torchaudio` (locally in the function) as it will create a global resampler, caching the filter coefficients (I got a big speedup from this previously).

Reply:
Done. Thanks for the tip!

Reply:
> Can you remove torchaudio imports from the global namespace into the local function you added? They tend to crash documentation builds last time I checked, because they tried to load some shared library that is not available in that env.

@pzelasko Do you have a log of the crash? We make sure that PyTorch is the only hard requirement. I am curious to see what it is.

Reply:
Sure, let me check it again in a spare moment. I think it was related to loading libsndfile.so which was not present in that system. It was also a while ago so maybe the current versions don't have that issue.